tests/integration/targets/synchronize/tasks/main.yml

```
@@ -227,3 +227,36 @@
   - directory_a/foo.txt
   - directory_a
   - directory_b
+- name: synchronize files with quiet option
+  synchronize:
+    src: '{{output_dir}}/foo.txt'
```

by default, some linters may complain about readability :)
```suggestion
    src: '{{ output_dir }}/foo.txt'
```